src/components/Favicon/Favicon.tsx

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable new-cap */
+import { Grid } from '@mui/material';
+import React from 'react';
+interface FaviconProps {
+  /* The link displayed in the UI */
+  displayLink: string;
+  /* The full URL */
+  link?: string;
```

Comment:
`link` isn't being used.
There just needs to be one prop really, the domain (e.g. "google.com"), from which this component can get the favicon and display it (if it isn't available it shouldn't display anything).